src/main/java/homeworks/jose/secondAssigment/ClassG.java

```diff
@@ -0,0 +1,14 @@
+package homeworks.jose.secondAssigment;
+
+public class ClassG extends Father implements MethodDE{
+    private boolean f;
+
+    @Override
+    public void methodA() {
+        super.methodA();
```

Comment:
there is no need for an override (and therefore the method) since the behavior of both methods are the same